packages/shared/src/hooks/useFeedSettings.ts

```diff
@@ -33,6 +33,12 @@ export type FeedSettingsReturnType = {
 
 let avoidRefresh = false;
 
+export const getHasAnyFilter = (feedSettings: FeedSettings): boolean =>
```

Comment:
In the current implementation this is only used in the test, so pretty sure we can remove it right?

Reply:
It is also being used in the `useFeedSettings`, just to be sure they are using the same process to fetch the correct value.

https://github.com/dailydotdev/apps/blob/3fdfe38302ba5fcde4b71fc3a6d44c390070c643/packages/shared/src/hooks/useFeedSettings.ts#L125